apps/api/src/routes/agents.routes.ts

```diff
@@ -168,25 +168,37 @@ export function configureAgentsRoutes(
    *                       type: object
    *                       description: stats on this agent's past performance
    *                       properties:
-   *                         competitionsCompleted:
+   *                         completedCompetitions:
+   *                           type: integer
+   *                         totalTrades:
+   *                           type: integer
+   *                         totalVotes:
    *                           type: integer
-   *                         provenSkills:
-   *                           type: array
-   *                           items:
-   *                             type: string
    *                         bestPlacement:
    *                           type: object
    *                           properties:
    *                             competitionId:
    *                               type: string
-   *                             postition:
+   *                             rank:
```

Comment:
just calling out that the `bestPlacement` object's field were altered to better align with existing terminology. (i.e., the `comps` app API client / types will need to adjust for this.)